app/models/accion_en_snapshot.rb

```diff
@@ -10,7 +10,7 @@ class AccionEnSnapshot < ActiveRecord::Base
 
   def self.crear_desde_dato(dato, i)
     create(accion: Accion.buscar_o_crear_desde_dato(dato),
-           posicion: dato['visitas'] || i + 1,
+           posicion: dato['posicion'] || i + 1,
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.